src/services/cleeng.profile.service.ts

```diff
@@ -0,0 +1,18 @@
+import { injectable } from 'inversify';
+
+import ProfileService from './profile.service';
+
+@injectable()
+export default class CleengProfileService extends ProfileService {
```

Comment:
Do we need this class? Can profile service just be undefined when cleeng is used?

Reply:
It can. I created it to be consistent with other integration-dependant services.

Reply:
Let's delete it. I think it should be expected that different app integrations sets may have totally different modules.